Source/WebKit/WebProcess/Extensions/Interfaces/WebExtensionAPITest.idl

```diff
@@ -82,4 +82,7 @@
     // Asserts the function does not throw an exception and the result promise is resolved.
     [NeedsScriptContext] any assertSafeResolve(any function, [Optional] DOMString message);
 
+    // Adds a test represented by a promise. If the promise is resolved, the test passes; if the promise is rejected, the test fails.
+    [NeedsScriptContext] any addTest(any function, DOMString testName);
```

Comment:
Since we can infer the test name from the function, and this order allows inline functions to flow better.
```suggestion
    [NeedsScriptContext] any addTest([Optional] DOMString testName, any function);
```